src/ctapipe/tools/optimize_event_selection.py

```diff
@@ -0,0 +1,216 @@
+"""Tool to generate selections for IRFs production"""
+
+import astropy.units as u
+from astropy.table import vstack
+
+from ..core import Provenance, Tool, traits
+from ..core.traits import AstroQuantity, Bool, Float, Integer, classes_with_traits, flag
+from ..irf import EventsLoader, Spectra
+from ..irf.optimize import CutOptimizerBase
+
+
+class IrfEventSelector(Tool):
+    name = "ctapipe-optimize-event-selection"
+    description = "Tool to create optimized cuts for IRF generation"
+
+    gamma_file = traits.Path(
+        default_value=None, directory_ok=False, help="Gamma input filename and path"
+    ).tag(config=True)
+
+    gamma_sim_spectrum = traits.UseEnum(
+        Spectra,
+        default_value=Spectra.CRAB_HEGRA,
+        help="Name of the pyirf spectra used for the simulated gamma spectrum",
```

Comment:
This is not the spectrum "used" it's the spectrum that is going to be used for weighting, right?

Reply:
Yes, I fixed the help strings in both tools accordingly.